### PR TITLE
Add CRT docker changes for release workflow

### PR DESCRIPTION
**Pull request by skpratt**
build: https://github.com/hashicorp/consul-k8s/actions/runs/5227978283

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -380,4 +426,26 @@ jobs:
...
+            fi
+            echo "Test PASSED"
+          version: ${{ env.version }}
+          target: ubi-fips # duplicate target to distinguish FIPS builds in CRT machinery
+          arch: ${{ matrix.arch }}
+          pkg_name: consul-k8s-control-plane_${{ env.version }}
+          bin_name: consul-k8s-control-plane
+          workdir: control-plane
+          tags: |
+            docker.io/hashicorp/${{ env.repo }}-control-plane:${{ env.version }}-ubi
```
Comment (zalimeni) [inline .github/workflows/build.yml:448]: I noticed this tag had `-fips` in it: https://github.com/hashicorp/consul-k8s/pull/2333/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R313 

Should this line too?

**Review by zalimeni — APPROVED** [review]
One question and generally curious which FIPS bits need vs. don't need `fips` in the name, but approving to unblock once that's resolved.

**Comment by zalimeni** [comment]
cc @wilkermichael

File: .github/workflows/build.yml (hunk 1 above)
Comment (skpratt) [reply .github/workflows/build.yml]: yes! thank you for spotting!!

**Review by wilkermichael — APPROVED** [review]

**Pull request merged**